keyboards/handwired/frame_48/keymaps/via/keymap.c

```diff
@@ -0,0 +1,88 @@
+    KC_TAB,     KC_A,     KC_S,       KC_D,       KC_F,     KC_G,     KC_MPLY,                    KC_MPLY,    KC_H,       KC_J,         KC_K,       KC_L,     KC_SCLN,  KC_QUOT,
+    KC_LSFT,    KC_Z,     KC_X,       KC_C,       KC_V,     KC_B,     KC_SPC,                     KC_SPC,     KC_N,       KC_M,         KC_COMM,    KC_DOT,   KC_UP,    KC_ENTER,
+    KC_MINUS,   KC_EQUAL, MO(_LOWER), KC_LCTL,    KC_RALT,  KC_LCMD,                                          KC_RCMD,    MO(_RAISE),   KC_SLASH,      KC_LEFT,  KC_DOWN,  KC_RIGHT
+),
+
+[_LOWER] = LAYOUT_frame_48(
+    KC_ESC,     KC_EXLM,  KC_AT,       KC_HASH,    KC_DLR,   KC_PERC,                                         KC_CIRC,    KC_AMPR,     KC_ASTR,    KC_LPRN,  KC_RPRN,  KC_BSPC,
+    KC_TRNS,    KC_LEFT,  KC_DOWN,     KC_UP,      KC_RIGHT, KC_TRNS,  KC_TRNS,                   KC_TRNS,    KC_LEFT,    KC_DOWN,     KC_UP,      KC_RIGHT, KC_TRNS,  KC_TRNS,
+    KC_TRNS,    KC_TRNS,  KC_TRNS,     KC_TRNS,    KC_TRNS,  KC_TRNS,  KC_TRNS,                   KC_TRNS,    KC_TRNS,    KC_TRNS,     KC_TRNS,    KC_TRNS,  KC_TRNS,  KC_TRNS,
+    KC_TRNS,    KC_TRNS,  MO(_LOWER),  KC_TRNS,    KC_TRNS,  KC_TRNS,                                         KC_TRNS,    MO(_RAISE),  KC_ENT,       KC_RALT,  KC_TRNS,  KC_TRNS
+),
+
+[_RAISE] = LAYOUT_frame_48(
+    KC_ESC,     KC_1,     KC_2,     KC_3,       KC_4,     KC_5,                                             KC_6,       KC_7,       KC_8,       KC_9,     KC_0,     KC_BSPC,
+    KC_TRNS,    KC_TRNS,  KC_TRNS,  KC_TRNS,    KC_TRNS,  KC_TRNS,  KC_TRNS,                      KC_TRNS,  KC_MINS,    KC_EQL,     KC_LCBR,    KC_RCBR,  KC_PIPE,  KC_GRV,
+    KC_TRNS,    KC_TRNS,  KC_TRNS,  KC_TRNS,    KC_TRNS,  KC_TRNS,  KC_TRNS,                      KC_TRNS,  KC_UNDS,    KC_PLUS,    KC_LBRC,    KC_RBRC,  KC_BSLS,  KC_TILD,
+    KC_TRNS,    KC_TRNS,  MO(_LOWER),  KC_TRNS,  KC_SPC,   KC_TRNS,                                         KC_TRNS,    MO(_RAISE), KC_ENT,     KC_RALT,  KC_TRNS,  KC_TRNS
+),
+[_ADJUST] = LAYOUT_frame_48(
+    KC_TRNS,    KC_TRNS,  KC_TRNS,     KC_TRNS,    KC_TRNS,  KC_TRNS,                                         KC_TRNS,    KC_TRNS,     KC_TRNS,    KC_TRNS,  KC_TRNS,  KC_TRNS,
+    KC_TRNS,    KC_TRNS,  KC_TRNS,     KC_TRNS,    KC_TRNS,  KC_TRNS,  KC_TRNS,                   KC_TRNS,    KC_TRNS,    KC_TRNS,     KC_TRNS,    KC_TRNS,  KC_TRNS,  KC_TRNS,
+    KC_TRNS,    KC_TRNS,  KC_TRNS,     KC_TRNS,    KC_TRNS,  KC_TRNS,  KC_TRNS,                   KC_TRNS,    KC_TRNS,    KC_TRNS,     KC_TRNS,    KC_TRNS,  KC_TRNS,  KC_TRNS,
+    KC_TRNS,    KC_TRNS,  MO(_LOWER),  KC_TRNS,    KC_TRNS,  KC_TRNS,                                         KC_TRNS,    MO(_RAISE),  KC_TRNS,    KC_TRNS,  KC_TRNS,  KC_TRNS
+),
+};
+
+layer_state_t layer_state_set_user(layer_state_t state) {
+    return update_tri_layer_state(state, _LOWER, _RAISE, _ADJUST);
+}
```

Comment:
```suggestion
```

You should use the `FN_MO13` and `FN_MO23` keycodes instead of MO, as those keycodes support the tri layer stuff.

Reply:
@drashna Do you happen to have an example of how to implement this? When I replace `MO(_LOWER)` keypresses with `FN_MO13` it ends up giving me an error saying `FN_MO13` is undeclared.